pkg/resources/credentials.go

```diff
@@ -295,6 +295,60 @@ func GetAWSCredentials(data CredentialsData) (AWSCredentials, error) {
 	return awsCredentials, nil
 }
 
+func GetAWSEKSCredentials(ctx context.Context, client ctrlruntimeclient.Client, cluster *kubermaticv1.ExternalCluster) (AWSCredentials, error) {
```

Comment:
I like those methods. Can you also move GKE here to have consistency

Reply:
sure

Reply:
thx

Reply:
done

Reply:
waiting for tests